itest/assertions.go

```diff
@@ -990,13 +990,15 @@ func AssertUniverseRootEquality(t *testing.T,
 	))
 }
 
+// AssertUniverseRoot makes sure the given universe root exists with the given
+// sum, either identified by the asset ID or group key.
 func AssertUniverseRoot(t *testing.T, client unirpc.UniverseClient,
-	sum int, assetID []byte, groupKey []byte) error {
+	sum int, assetID []byte, groupKey []byte) {
 
 	bothSet := assetID != nil && groupKey != nil
 	neitherSet := assetID == nil && groupKey == nil
 	if bothSet || neitherSet {
-		return fmt.Errorf("only set one of assetID or groupKey")
+		t.Fatalf("only set one of assetID or groupKey")
 	}
```

Comment:
If we're not going to return an error at this point then we can just use `require.True` and populate the message arg.